src/get-dart/index.md

```diff
@@ -39,6 +26,10 @@ NOTE to editors: Keep the zip file link as the last thing in the paragraph,
 so it's easy to find (but not more tempting than package managers).
 {% endcomment %}
 
+*Note*: The Flutter SDK includes the full Dart SDK,
+and has Dart's [`dart`](/tools/dart-tool) command-line interface
+in it's `bin` folder.
+
```

Comment:
it's -> its, but overall I think it's good to say this sooner rather than later.

```suggestion
```